Add namespaces to files

GaryJones opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
Add namespaces to several files, and adjust references. See #396.

This PR also moves the missed integration tests around - fixes #425.

This PR does NOT include a namespace on the `Parsely` class file, as the effects are too-wide reaching (think `@covers` and `@uses` annotations in tests). This will come in a future PR.

The PR also does NOT move the widget class into its own directory and namespace. I was able to add the widget when I had done, but was getting errors, even after running `npm run build`. Can be handled in a future PR.

## Motivation and Context
Namespaces allow long class names to be shortened as classes are logically grouped together. Some files already had namespaces, so this adds them to all but the `Parsely` class file and the view files.

## How Has This Been Tested?
Run the CI, and visually checked.

BREAKING CHANGE.

GaryJones committed:
Add namespaces to some files

See #396.

Also move integration tests around - fixes #425.

pauarge commented on src/class-parsely-recommended-widget.php:
```diff
@@ -12,10 +12,7 @@
 /**
- * This is the class for the recommended widget
- *
- * @category   Class
- * @package    Parsely_Recommended_Widget
```
Why are we removing this kind of annotations?

GaryJones replied:
Because they aren't valid for class and function/method DocBlocks. 

`@category` is deprecated / removed from the draft list of [PHP DocBlock tags](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md), as `@package` can accept multiple levels (`@subcategory` is also deprecated / removed for the same reason).

@package is only typically useful until namespaces are used anyway, so we'd keep a correct `@package` in the file-level DocBlock (it's wrong in this file), and that's the only single `@package` tag that should be in a file.

pauarge commented on tests/UI/PluginsActionsTest.php:
```diff
@@ -1,47 +0,0 @@
-<?php
```
Are we sure to delete this file?

GaryJones replied:
Yes, because it already exists at [`tests/Integrations/UI/PluginsActionsTest.php`](https://github.com/Parsely/wp-parsely/blob/develop/tests/Integration/UI/PluginsActionsTest.php).

I'd added it as a separate PR recently because I thought it was missing after I messed up a rebase, but I just wasn't looking for it in the right place.

pauarge commented:
👋 @GaryJones Could you check this PR again? Tests are failing.

GaryJones committed:
Namespaces: Fix constants

GaryJones committed:
Tests: Fix test after default changed

Default for whether to show the "Open In" changed in #433, but the test shouldn't have been making assumptions like that in the first place.

pauarge committed:
Merge branch 'develop' into fix/add-remaining-namespaces

GaryJones committed:
Fix path to widget file
